### PR TITLE
BACK-1312: use currentFee for RamsesV2 (UniV3 fork) pools

### PR DESCRIPTION
**Pull request by aburkut**

### CODE REVIEW

File: src/dex/uniswap-v3/uniswap-v3.ts [hunk 1]
```diff
@@ -287,25 +287,29 @@ export class UniswapV3
...
+          token0,
+          token1,
+          this.logger,
+          this.cacheStateKey,
+          this.config.initHash,
       );
 
     try {
-      await pool.initialize(blockNumber, {
+      await pool!.initialize(blockNumber, {
```
Comment (Verisana) [inline src/dex/uniswap-v3/uniswap-v3.ts:312]: I don't get, why you need all these exclamation marks? I removed and it compiles 🤔

Comment (aburkut) [reply src/dex/uniswap-v3/uniswap-v3.ts:312]: @Verisana Strange, for some reasons I was getting ts errors.

Comment (Verisana) [reply src/dex/uniswap-v3/uniswap-v3.ts:312]: Do you still get errors or now compiles?

Comment (aburkut) [reply src/dex/uniswap-v3/uniswap-v3.ts]: @Verisana Removed exclamation marks, I don't know why for some reasons my IDE showed ts errors for these cases but now it's OK already.

**Review by Verisana — APPROVED** [review]

**Review by paraschiv-razvan — APPROVED** [review]

**Review by JackieJoo — APPROVED** [review]

**Pull request merged**